Makefile

```diff
@@ -169,30 +170,33 @@ k8s-fv-test: calico/felix run-k8s-apiserver k8sfv/k8sfv.test
 	calico/felix \
 	/bin/sh -c "for n in 1 2; do calico-felix; done"
 	sleep 1
-	K8S_IP=`$(GET_CONTAINER_IP) k8sfv-apiserver` && \
-	docker exec k8sfv-felix /testcode/k8sfv/k8sfv.test -ginkgo.v https://$${K8S_IP}:6443
+	K8S_IP=`$(GET_CONTAINER_IP) $(K8SFV_PREFIX)-apiserver` && \
+	docker exec $(K8SFV_PREFIX)-felix /testcode/k8sfv/k8sfv.test -ginkgo.v https://$${K8S_IP}:6443
+	docker rm -f $(K8SFV_PREFIX)-felix
+	docker rm -f $(K8SFV_PREFIX)-apiserver
+	docker rm -f $(K8SFV_PREFIX)-etcd
 
 run-k8s-apiserver: stop-k8s-apiserver run-etcd
-	ETCD_IP=`$(GET_CONTAINER_IP) k8sfv-etcd` && \
+	ETCD_IP=`$(GET_CONTAINER_IP) $(K8SFV_PREFIX)-etcd` && \
 	docker run --detach \
-	  --name k8sfv-apiserver \
+	  --name $(K8SFV_PREFIX)-apiserver \
 	gcr.io/google_containers/hyperkube-amd64:v$(K8S_VERSION) \
 		  /hyperkube apiserver --etcd-servers=http://$${ETCD_IP}:2379 \
 		  --service-cluster-ip-range=10.101.0.0/16 -v=10
 
 stop-k8s-apiserver: stop-etcd
-	@-docker rm -f k8sfv-apiserver
+	@-docker rm -f $(K8SFV_PREFIX)-apiserver
```

Comment:
Won't think now stop a randomly-named API server?  That doesn't feel so useful.  Maybe we should pass in the prefix as a variable or default to the jenkins job number if defined?